CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ This document describes changes between each past release.
 **Internal changes**
 
 - Fix documentation of errors codes (fixes #766)
+- ``kinto.id_generator`` was removed from documentation since it does not
```

Comment:
I spent some time but gave up. Don't hesitate to try but I think it's harder than expected :)